features/upgrade/api_auth/upgrade.feature

```diff
@@ -0,0 +1,52 @@
+    When I run the :get admin command with:
+      | resource | clusterroles.rbac |
+      | o        | yaml              |
+    Then the output is parsed as YAML
+    And the output should contain:
+      | rbac.authorization.kubernetes.io/autoupdate: "true" |
+    And the output should not contain:
+      | rbac.authorization.kubernetes.io/autoupdate: "false" |
+    Given as admin I successfully patch resource "clusterrole.rbac/system:build-strategy-docker" with:
+      | {"rules": [{"apiGroups": ["","build.openshift.io"],"resources": ["builds/docker","builds/optimizeddocker"],"verbs": [ "get" ]}] } |
+    When I run the :get admin command with:
+      | resource      | clusterroles.rbac            |
+      | resource_name | system:build-strategy-docker |
+      | o             | yaml                         |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] == "get"
+    Given as admin I successfully patch resource "clusterrolebinding.rbac/system:oauth-token-deleters" with:
+      | {"subjects":[{"apiGroup": "rbac.authorization.k8s.io","kind": "Group","name": "system:authenticated"}]} |
+    When I run the :get admin command with:
+      | resource      | clusterrolebinding.rbac     |
+      | resource_name | system:oauth-token-deleters |
+      | o             | yaml                        |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['subjects'][0]['name'] == "system\:authenticated"
+
+  # @author kewang@redhat.com
+  # @case_id OCP-19470
+  @upgrade-check
+  @admin
+  Scenario: Default RBAC role, rolebinding, clusterrole and clusterrolebinding without any missing after upgraded
+    When I run the :get admin command with:
+      | resource      | clusterroles.rbac            |
+      | resource_name | system:build-strategy-docker |
+      | o             | yaml                         |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['rules'][0]['verbs'][0] == "get"
+    And the expression should be true> @result[:parsed]['rules'][1]['verbs'][0] == "create"
+    And the expression should be true> @result[:parsed]['rules'][2]['verbs'][0] == "create"
+    And the expression should be true> @result[:parsed]['rules'][3]['verbs'][0] == "create"
+    And the expression should be true> @result[:parsed]['rules'][4]['verbs'][0] == "create"
+    When I run the :get admin command with:
+      | resource      | clusterrolebinding.rbac     |
+      | resource_name | system:oauth-token-deleters |
+      | o             | yaml                        |
+    Then the output is parsed as YAML
+    And the expression should be true> @result[:parsed]['subjects'][0]['name'] == "system\:authenticated"
+    And the expression should be true> @result[:parsed]['subjects'][1]['name'] == "system\:unauthenticated"
```

Comment:
Same minor issue, `\` is not needed.

Reply:
'\' is needed here, otherwise test will run into this.

Reply:
Fixed.